src/content/docs/en/reference/directives-reference.mdx

```diff
@@ -225,6 +225,8 @@ The `is:inline` directive means that `<style>` and `<script>` tags:
 
 :::caution
 The `is:inline` directive is implied whenever any attribute other than `src` is used on a `<script>` or `<style>` tag.
+
```

Comment:
```suggestion
```
Just making this all one paragraph so the exception doesn't seem so far removed from the earlier statement!